pkg/webhook/resources/virtualmachine/validator.go

```diff
@@ -239,6 +242,22 @@ func (v *vmValidator) checkVMBackup(vm *kubevirtv1.VirtualMachine) error {
 	return nil
 }
 
+func (v *vmValidator) checkTerminationGracePeriodSeconds(vm *kubevirtv1.VirtualMachine) error {
+	TerminationGracePeriodSeconds := vm.Spec.Template.Spec.TerminationGracePeriodSeconds
```

Comment:
non-constant variable within local scope starts with lower case conventionally.

Reply:
Thanks, I will modify it.